docs/profiling-workflow-dotnet-runtime.md

```diff
@@ -2,41 +2,42 @@
 
 ## Table of Contents
 
-- [Introduction](#Introduction)
-- [Prerequisites](#Prerequisites)
-  - [Build](#Build)
-  - [Repro](#Repro)
-  - [Project Settings](#Project-Settings)
-- [Visual Studio Profiler](#Visual-Studio-Profiler)
+- [Profiling workflow for dotnet/runtime repository](#profiling-workflow-for-dotnetruntime-repository)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Prerequisites](#prerequisites)
+  - [Build](#build)
+  - [Repro](#repro)
+  - [Project Settings](#project-settings)
+  - [Visual Studio Profiler](#visual-studio-profiler)
   - [dotnet](#dotnet)
-  - [CPU Usage](#CPU-Usage)
-  - [CoreRun](#CoreRun)
-  - [Allocation Tracking](#Allocation-Tracking)
-- [PerfView](#PerfView)
-  - [CPU Investigation](#CPU-Investigation)
-  - [Filtering](#Filtering)
-  - [Analyzing the Results](#Analyzing-the-Results)
-  - [Viewing Source Code](#Viewing-Source-Code)
-  - [Identifying Regressions](#Identifying-Regressions)
-- [VTune](#VTune)
-  - [When to use](#When-to-use)
-  - [Identifying Hotspots](#Identifying-Hotspots)
-  - [Troubleshooting](#Troubleshooting)
-  - [Code](#Code)
-  - [Skids](#Skids)
-  - [Linux](#Linux)
-- [PerfCollect](#PerfCollect)
-  - [Preparing Your Machine](#Preparing-Your-Machine)
-  - [Preparing Repro](#Preparing-Repro)
-  - [Collecting a Trace](#Collecting-a-Trace)
-  - [Analyzing the Trace](#Analyzing-the-Trace)
-
-
-# Introduction
+  - [CPU Usage](#cpu-usage)
+  - [CoreRun](#corerun)
+  - [Allocation Tracking](#allocation-tracking)
+  - [PerfView](#perfview)
+  - [CPU Investigation](#cpu-investigation)
+  - [Filtering](#filtering)
+  - [Analyzing the Results](#analyzing-the-results)
+  - [Viewing Source Code](#viewing-source-code)
+  - [Identifying Regressions](#identifying-regressions)
+  - [VTune](#vtune)
+  - [When to use](#when-to-use)
+  - [Identifying Hotspots](#identifying-hotspots)
+  - [Troubleshooting](#troubleshooting)
+  - [Code](#code)
+  - [Skids](#skids)
+  - [Linux](#linux)
+  - [PerfCollect](#perfcollect)
+  - [Preparing Your Machine](#preparing-your-machine)
```

Comment:
Why did we change headers from being sub-headings?

Reply:
Ah. An extension I had installed automatically fixes up the TOC to be in-line with the headers below. I think fixing the headings is a better idea anyway. I'll do that.